keras/src/backend/torch/math.py

```diff
@@ -203,6 +203,15 @@ def fft2(x):
     return complex_output.real, complex_output.imag
 
 
+def ifft2(x):
```

Comment:
https://pytorch.org/docs/stable/generated/torch.fft.ifft2.html

Reply:
updated to use built-in version